lucet-runtime/lucet-runtime-internals/src/instance/execution.rs

```diff
@@ -1,20 +1,23 @@
 //! The `execution` module contains state for an instance's execution, and exposes functions
```

Comment:
All of the changes in the top-level module documentation are worth proofreading carefully. I tried to make sure this was all updated properly, but would love a second set of eyes here.